### PR TITLE
FAI-11601 | Added run_mode param for GitHub source

### PR DESCRIPTION
**Pull request by dbruno21**
## Description

- Added run_mode param for GitHub source
  - CopilotEvaluationApp mode streams already finished

## Type of change
- [ ] Bug fix
- [X] New feature
- [ ] Breaking change

### CODE REVIEW

**Review by ypc-faros — APPROVED** [review]

File: sources/github-source/src/streams/common.ts [hunk 1]
```diff
@@ -8,6 +8,48 @@ export type OrgStreamSlice = {
   org: string;
 };
 
+export enum RunMode {
+  MarketplaceApp = 'MarketplaceApp',
```
Comment (chalenge) [inline sources/github-source/src/streams/common.ts:12]: this should be FarosMarketplaceApp

File: sources/github-source/resources/spec.json [hunk 2]
```diff
@@ -117,6 +117,19 @@
...
         }
+      },
+      "run_mode": {
+        "order": 6,
+        "title": "Run Mode",
+        "description": "Run mode for selecting the usage of the connector",
+        "type": "string",
+        "default": "Standard",
+        "enum": [
+          "MarketplaceApp",
```
Comment (chalenge) [inline sources/github-source/resources/spec.json:128]: FarosMarketplaceApp. Well actually what is the name of the app in the marketplace? We should use that

**Review by chalenge — COMMENTED** [review]
Let's use correct name for `MarketplaceApp`

File: sources/github-source/resources/spec.json (hunk 2 above)
Comment (dbruno21) [reply sources/github-source/resources/spec.json]: updated

File: sources/github-source/src/streams/common.ts (hunk 1 above)
Comment (dbruno21) [reply sources/github-source/src/streams/common.ts]: updated

**Pull request merged**